content/post/050-ipfs-camp-recap.md

```diff
@@ -0,0 +1,87 @@
+> Get blown away by huge Sci-Fi experiments, soon to be seen in a Network close to you
+
+![](https://ipfs.io/ipfs/QmUiucP3oRVnqvuM6hnrF9D5H6tRSeVhyWk6ggkhf4ebEH)
+
+Catch up on all the interviews at [📼](https://github.com/ipfs/camp#-sci-fi-fair)
+
+#### [**🧩 Unconf**](https://github.com/ipfs/camp#-unconf)
+> Gather folks around shared topics of interest and discuss collaborations.
+
+![](https://ipfs.io/ipfs/QmUiucP3oRVnqvuM6hnrF9D5H6tRSeVhyWk6ggkhf4ebEH)
+
+Catch up on all the artefacts at [📒](https://github.com/ipfs/camp#-unconf)
+
+### IPFS Camp `Success` is `defined` by the number of `friendships`, `conversations`, `collaborations` and `new projects` that get started.
+
+And it was a blast. Here is a quick video recap as a souvenir for everyone that was there and a postcard for everyone that didn't get a chance this time:
+
+[![](img/060-ipfs-camp-recap/ipfs-camp-youtube-preview)](https://youtu.be/kc_dxO-V8YM)
+
+However, the IPFS Camp spirit doesn't stop here, now it is all about continuing the dialog that got started and taking all the learnings and materials to local IPFS Communities, showing and teaching more folks how to build on the P2P Web. All the content produced can be found on the IPFS Camp Repo -- http://github.com/ipfs/camp. Soon we will be posting all the recordings as well.
+
+Last, but definitely not the least, a huge THANK YOU ❤️ to everyone that contributed to the event. IPFS Camp wouldn't be possible if it werent for:
+
+- **160 Attendees** coming from over **22 Countries**!
+- [**67 Content Creators**](https://camp.ipfs.io/team) that produced all the training and talk material for the camp attendees
+- An outstanding [Production Team](https://camp.ipfs.io/team) that created the phenomenal experienced for everyone.
+- The over 4000 IPFS Community members that have been building this project with us since the beginning.
+
+[![](https://media.giphy.com/media/mGW5xq4SwlqnSTxwtb/giphy.gif)](img/060-ipfs-camp-recap/camp-gif.gif)
+
+In name of the whole IPFS Project, thank you all for being part of this project, such an ambitious goal couldn't be achieved without all your energy, passion and will to do the right thing.
+
+David Dias & IPFS Team
+
+`Extra:` As a closing gift, here are some random interesting facts about Camp:
+
+- 😸 IPFS Camp was the 1st time the Lazer Cat bot became [**P2P Lazer Cat bot**](https://github.com/gorhgorh/ipfscatremote)
+- 🍦 All you can eat fruit ice cream made all attendees very happy.
+- 👋🏽 Textile created a [**P2P Tag Game**](https://github.com/textileio/advanced-react-native-boilerplate/blob/ipfs-tag/README.md). I believe the last person to be "it" is still looking the next person to tag, so look out at next events!
+- 👕 The first **SWAG Trading Post** was established, an eco-responsible way of attendees picking the item of swag they love the most or that they can trade for others, avoiding the regular Conference waste of ETOOMANYBELLSANDWHISTLES.
```

Comment:
What is "ETOOMANYBELLSANDWHISTLES"? I don't understand this reference...

Reply:
I just replaced it with a generic alternative

Reply:
The uppercase glued sentence comes from Unix/Posix errors (e.g. ENOENT, ENOTDIR, etc) and the "bells and whistles" is an expression used to describe [superfluous items that are not required for something to function properly](https://idioms.thefreedictionary.com/bells+and+whistles).

In my head, it made sense, however, all good if you decide to remove it :)